apps/alphabet_mobile/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
```

Comment:
you might look into best practices around source control and the .png files for react native apps